CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.17.10.0]
+### Added
+- Caching for diff and content app summary
+- Restriction of hash repair for a large number of files
```

Comment:
Should be under `Changed` category as `Disable checking hashes for repairing after failed installation if files count is over 10000`.